core/src/epicli/data/common/defaults/configuration/applications.yml

```diff
@@ -124,3 +124,29 @@ specification:
         # Specify content for pgpool.conf
         # TODO: To be implemented
         # Example: https://github.com/CrunchyData/crunchy-containers/blob/master/examples/kube/pgpool/configs/pgpool.conf
+
+  - name: pgbouncer
+    enabled: yes
+    #https://github.com/brainsam/pgbouncer/tree/release/1.12
+    image_path: brainsam/pgbouncer:1.12
+    namespace: pgbouncer
+    service:
+      name: pgbouncer
+      port: 5432
+    replicas: 2
+    pgbouncer:
+      env:
+        target_address: pgpool.pgpool.svc.cluster.local #pgpool service name
+        target_port: "5432"
```

Comment:
I think numbers don't need quotation, all values are quoted in the configMap template.